### PR TITLE
Added internal option

### PR DESCRIPTION
**Pull request by daniel-jeremia**
- Added internal option to use Datasaur ECR

### CODE REVIEW

File: charts/datasaur/templates/main/backend-worker-background-deployment.yaml [hunk 1]
```diff
@@ -26,7 +26,11 @@ spec:
         name: {{ .Release.Name }}-mount-file
       {{- end }}
       containers:
+{{- if .Values.image.internal }}
         - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}datasaur/backend-on-premise:{{ .Values.image.dockerTag }}
```
Comment (tonoman3g) [inline charts/datasaur/templates/main/backend-worker-background-deployment.yaml]: you forgot to change this path

File: charts/datasaur/templates/main/backend-worker-critical-deployment.yaml [hunk 2]
```diff
@@ -26,7 +26,11 @@ spec:
         name: {{ .Release.Name }}-mount-file
       {{- end }}
       containers:
+{{- if .Values.image.internal }}
         - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}datasaur/backend-on-premise:{{ .Values.image.dockerTag }}
```
Comment (tonoman3g) [inline charts/datasaur/templates/main/backend-worker-critical-deployment.yaml]: you forgot to change this path

File: charts/datasaur/templates/main/backend-worker-external-deployment.yaml [hunk 3]
```diff
@@ -26,7 +26,11 @@ spec:
         name: {{ .Release.Name }}-mount-file
       {{- end }}
       containers:
+{{- if .Values.image.internal }}
         - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}datasaur/backend-on-premise:{{ .Values.image.dockerTag }}
```
Comment (tonoman3g) [inline charts/datasaur/templates/main/backend-worker-external-deployment.yaml]: you forgot to change this path

Comment (daniel-jeremia) [reply charts/datasaur/templates/main/backend-worker-external-deployment.yaml]: my bad, fixed @tonoman3g

**Review by tonoman3g — APPROVED** [review]

**Pull request merged**